eZ/Bundle/EzPublishCoreBundle/Tests/DependencyInjection/EzPublishCoreExtensionTest.php

```diff
@@ -891,4 +891,39 @@ public function testRichTextConfiguration()
             $this->container->getParameter($this->extension::RICHTEXT_CUSTOM_TAGS_PARAMETER)
         );
     }
+
+    public function testUrlAliasConfiguration()
+    {
+        $configuration = [
+            'transformation' => 'urlalias_lowercase',
+            'separator' => 'dash',
+            'transformation_groups' => [
+                'urlalias' => [
+                    'commands' => [
+                        'ascii_lowercase',
+                        'cyrillic_lowercase',
+                    ],
+                    'cleanup_method' => 'url_cleanup',
+                ],
+                'urlalias_compact' => [
+                    'commands' => [
+                        'greek_normalize',
+                        'exta_lowercase',
+                    ],
+                    'cleanup_method' => 'compact_cleanup',
+                ],
+            ],
+        ];
+        $this->load([
+            'url_alias' => [
+                'slug_converter_config' => $configuration,
+            ],
+        ]);
+        $this->assertTrue($this->container->hasParameter('ezpublish.url_alias.slug_converter_config'));
```

Comment:
Nitpick: This assertion is redundant. If a parameter is not defined  test will fail because of `\InvalidArgumentException` 😉